common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/VroPackageStore.java

```diff
@@ -111,8 +156,12 @@ public List<Package> importAllPackages(List<Package> vroPackages, boolean dryrun
 
     @Override
     public Package exportPackage(Package vraPackage, boolean dryrun) {
-        logger.info(String.format(PackageStore.PACKAGE_EXPORT, vraPackage));
-        Package pkg = restClient.exportPackage(vraPackage, dryrun);
+		logger.info(String.format(PackageStore.PACKAGE_EXPORT, vraPackage));
+
+		boolean exportConfigAttributeValues = false;
```

Comment:
@ivo-kotev in the **importAllPackages** method used for the backup the values of **exportConfigAttributeValues** and **exportConfigSecureStringValues** are true.
However on lines 161, 162, 175 and 176 I keep them false because this is a different pipeline from the backup and I keep its old behaviour to get their values from configuration(line 222 in RestClientVro.java)